### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Abdo9826**
- Create a directory called components.

- Inside components, create a new Calculator.js file.

- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- 
- Once the Calculator is ready, import it and use it in your main component (App.js).

- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      <div className="calculator card">
+        <input
+          type="text"
+          className="calculator-screen z-depth-1"
+          value=""
+          disabled
+        />
+
+        <div className="calculator-keys">
+          <button type="button" className="operator btn btn-info" value="+">
```
Comment (AmaduKamara) [inline src/components/Calculator.js]: - [ ] Kindly make this component a class-based component. For this project, you're to write your components in class-based so later you'll refactor to functional components. Please refactor this functional component to a class-based component 👍

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import App from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
 );
```
Comment (AmaduKamara) [inline src/index.js]: - [ ] What you did here actually works but it's not the standard way of react library. It'll be good if you have an App.js component, into which you will call the Calculator.js component and then the App.js component is then called inside of the index.js component. But currently, you named your Calculator.js component call inside of index as App and then passed it in index.js. That is not the required way. 
- [ ] Kindly recreate the App.js component, make it a class-based as well, then call the Calculator.js inside this App.js, and in your index.js file, you can then call the App.js component. 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @Abdo9826,

AlonzoLeroneThisCloseGIF (3)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Correct use of Gitflow
✔️ No linter error
✔️ README is okay
✔️ PR description is okay 👍

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly design your app exactly as the design is given, and have all calculation buttons the same.

**Your Design**
image
 
**The original design**
image

- [ ] Please work on it 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by t-yanick — APPROVED** [review]
Hi @Abdo9826, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**